MainModule/Server/Commands/HeadAdmins.lua

```diff
@@ -446,5 +446,25 @@ return function(Vargs, env)
 				end
 			end;
 		};
+																		
+		bones = {
+			Prefix = Settings.Prefix;
+			Commands = {"bones"};
+			Args = {};
+			Description = "bones (very destruct)";
+			AdminLevel = "HeadAdmins";
+			Filter = true;
+			IsCrossServer = false;
+			Function = function(plr,args)
+				while wait() do
+					local bone = Instance.new("Part",workspace)
```

Comment:
You should put table.insert(Variables.Objects, bone)

So adonis can automatically clear these

Reply:
Yeah~~, well actually no we want the bones to fill up the server and squeeze em players to dea—~~